src/components/MultiHopTrade/components/TradeInput/components/MaxSlippage.tsx

```diff
@@ -108,8 +112,6 @@ export const MaxSlippage: React.FC<MaxSlippageProps> = ({
   const divider = useMemo(() => <Divider borderColor='border.base' />, [])
 
   const tooltipBody = useCallback(() => {
-    if (!shouldShowSlippageMinAmount)
-      return <Text color='text.subtle' translation='trade.tooltip.slippage' />
```

Comment:
That was derp - the var was actually called `shouldShowSlippageMinAmount`, but when set to false, we... still showed slippage min amount, and `Your transaction will revert if the price changes unfavorably by more than this percentage.`, wut:

 <img width="460" alt="image" src="https://github.com/user-attachments/assets/ff72de44-ffa6-4734-9c95-f75e9f4d31d9" />

In addition to hiding the percentage for unknown auto slippage, we also ensure that we now hide the tooltip in that case in l.171 below, meaning we'll never see this guy (as it should be!)